alacritty_terminal/src/ansi.rs

```diff
@@ -25,62 +25,49 @@ use vte;
 use crate::term::color::Rgb;
 
 // Parse color arguments
```

Comment:
This comment is outdated/out of place.

Reply:
[Moved](https://github.com/rbong/alacritty/blob/cdbcd8357fc2955c011785ed02bd93b386509561/alacritty_terminal/src/ansi.rs#L31-L32)